src/app/services/voting.service.js

```diff
@@ -124,7 +125,11 @@ class Voting {
 
         let account;
         if (common.isHW) {
-            account = new TrezorAccount(this._Wallet.currentAccount.address, this._Wallet.currentAccount.hdKeypath);
+            if (this._Wallet.algo == "trezor") {
+                account = new TrezorAccount(this._Wallet.currentAccount.address, this._Wallet.currentAccount.hdKeypath);
+            } else if (this._Wallet.algo == "ledger") {
+                account = {};
```

Comment:
not setting the current account for voting?

Reply:
No need re-create the current for Ledger account here